src/views/CrontabForm/CrontabForm.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useState } from "react";
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  ActionGroup,
+  Button,
+  PageSection,
+  Title,
+  Alert,
+  NumberInput,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+} from "@patternfly/react-core";
+import {
+  CRONTAB_APIGROUP_VERSION,
+  CRONTAB_APIVERSION,
+  CRONTAB_KIND,
+  CRONTAB_APIGROUP,
+} from "src/const";
+import {
+  useK8sModel,
+  k8sCreate,
+  useActiveNamespace,
+} from "@openshift-console/dynamic-plugin-sdk";
+import { useNavigate } from "react-router-dom-v5-compat";
+import { useCronTabTranslation } from "@crontab-utils/hooks/useCronTabTranslation";
+import { CronTabKind } from "@crontab-model";
+
+export const CronTabForm: React.FC = () => {
+  const [model] = useK8sModel({
+    version: CRONTAB_APIVERSION,
+    kind: CRONTAB_KIND,
+    group: CRONTAB_APIGROUP,
+  });
+  const [name, setName] = useState("");
+  const [cronSpec, setCronSpec] = useState("");
+  const [image, setImage] = useState("");
+  const [replicas, setReplicas] = useState(1);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState("");
+  const navigate = useNavigate();
+  const { t } = useCronTabTranslation();
+  const [namespace] = useActiveNamespace();
+
+  const onReplicasMinus = () => {
+    setReplicas((currentReplicas) => currentReplicas - 1);
+  };
+  const onReplicasPlus = () => {
+    setReplicas((currentReplicas) => currentReplicas + 1);
+  };
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    setLoading(true);
+    setError("");
+
+    const data: CronTabKind = {
+      apiVersion: CRONTAB_APIGROUP_VERSION,
+      kind: CRONTAB_KIND,
+      metadata: {
+        name,
+        namespace,
+      },
+      spec: {
+        cronSpec,
+        image,
+        replicas,
+      },
+    };
+
+    k8sCreate({ model, data })
+      .then(() => {
+        setLoading(false);
+        navigate(`/k8s/ns/${namespace}/stable.example.com~v1~CronTab`);
+      })
+      .catch((err) => {
+        setLoading(false);
+        setError(t("Error creating CronTab: {{err}}", { err: err.toString() }));
+      });
+  };
+  return (
+    <>
```

Comment:
nit, remove React.Fragment. This is only needed to wrap return values with multiple elements.
```suggestion
```